src/Parallel/Callback.hpp

```diff
@@ -37,9 +37,10 @@ class SimpleActionCallback : public Callback {
  public:
   WRAPPED_PUPable_decl_template(SimpleActionCallback);  // NOLINT
   SimpleActionCallback() = default;
-  SimpleActionCallback(Proxy proxy, Args&&... args)
+  // NOLINTNEXTLINE(google-explicit-constructor)
+  SimpleActionCallback(Proxy proxy, std::decay_t<Args>... args)
```

Comment:
Why do you need `decay_t` all over the place in this class?

Reply:
Well `Args...` keeps all the `const &` so I had to remove those for the declaration of the member tuple so it wasn't a tuple of references, but I left everything else as `Args...`. However, my test failed with that because a `DataVector` I was using to test actually got moved into this callback instead of copied (not `std::move` from the calling code, but from the `std::move` one line below) and this caused subsequent tests to fail. When I did `std::decay_t<Args>...` in the constructor, everything worked fine.

Reply:
I see.  I think you're right and this is the correct way to do it here.

Optionally, you might want to add `decay_t` to the calling code as well, just to keep the number of equivalent types the compiler has to deal with down.  Probably not many instantiations of this class though, so not very important.

Reply:
Other than the test, the calling code (in the FoT are ready functions) is usually `std::forward<Args>(args)...`. So I'll leave that as is.